src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XpathUtil.java

```diff
@@ -36,6 +36,9 @@
  */
 public final class XpathUtil {
 
+    public static final String IT_RESOURCES_XPATH_FILTER_PATH =
+            "src/it/resources/org/checkstyle/suppressionxpathfilter";
```

Comment:
Can't the test class extend something like https://github.com/checkstyle/checkstyle/blob/master/src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java and then call `getPath("")`?
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/AbstractPathTestSupport.java#L57